content/doc/book/blueocean/dashboard.adoc

```diff
@@ -19,105 +19,78 @@ endif::[]
-the initial part of the commit hash, and the last run time. This list also 
-includes clickable icons to run or re-run the item on the repository
-branch indicated.
+This list provides key information and actions for a core subset of your accessible items in the <<pipelines-list>>.
+This key information includes the current link:#run-status[run status] for an item and its repository's branch, the name of the branch, the initial part of the commit hash, and the last run time. 
+This list also includes options to run or re-run the item on the indicated branch.
 
-You should only add an item (or one of the repository's specific branches) to
-your Favorites list if you need to regularly examine that item's branch. Adding
-an item's specific branch to your Favorites list can be done
-through the item's link:../activity[Activity View].
+You should only add an item or branch to your Favorites list if it needs regular monitoring.
+Adding an item's specific branch to your Favorites list can be done through the item's link:../activity[Activity View].
 
-Blue Ocean automatically adds branches or PRs to this list when they contain a
-run that has changes authored by the current user.
+Blue Ocean automatically adds branches and PRs to this list when they include a run that contains any modifications you have performed.
 
-You can also manually remove items from your Favorites list by clicking on the
-solid "&#9733;" in this list. When the last item is removed from this list, the
-list is removed from the interface.
-
-Clicking on an item in the Favorites list will open the
-link:../pipeline-run-details[Pipeline run details] for the latest run on the
-repository branch or PR indicated.
+You can also manually remove items from your Favorites list by deselecting the solid &#9733; in this list.
+When there are no longer items in this list, the list is removed from the Dashboard.
+Selecting the favorite &#9733; for any item will bring the list back to your dashboard.
 
+Selecting an item in the Favorites list will open the link:../pipeline-run-details[Pipeline run details] for the latest run on the branch or PR indicated.
 
 [[pipeline-health]]
 === Health icons
 
-Blue Ocean represents the overall health of a Pipeline/item or one of its
-repository's branches using weather icons, which change depending on the number
-of recent builds that have passed.
+Blue Ocean represents the overall health of a Pipeline/item or one of its branches using weather icons. 
+These icons change depending on the number of recent builds that have passed.
 
-Health icons on the Dashboard represent overall Pipeline health, whereas the
-health icons in the link:../activity#branches[Branches tab of the Activity View]
-represent the overall health for each branch.
+Health icons on the Dashboard represent overall Pipeline health, whereas the health icons in the link:../activity#branches[Branches tab of the Activity View] represent the overall health for each branch.
```

Comment:
Pointing out the last instance of "Dashboard" in case you want to do something with it. ;)